Tools/Cluster.py

```diff
@@ -17,7 +17,9 @@
 from Tools import *
 
 
```

Comment:
The  clustering should be done first ,  then we sort the pixel according to PFT co-occurrence.  Otherwise, we might remove a whole cluster by chance in case a cluster tend to occur in absence of co-occurence other PFTs 

this seems to be the case, this is commentary is only to make sure it is like I undersood

Reply:
Yes I can confirm that is the case